src/xrc/xh_choicbk.cpp

```diff
@@ -123,9 +116,25 @@ wxObject *wxChoicebookXmlHandler::DoCreateResource()
         m_choicebook = nb;
         bool old_ins = m_isInside;
         m_isInside = true;
+        wxVector<wxBitmapBundle> old_images = m_bookImages;
+        m_bookImages.clear();
```

Comment:
This would be slightly more efficient (also below and at the of the function):
```suggestion
        wxVector<wxBitmapBundle> old_images;
        swap(m_bookImages, old_images);
```